eden/fs/config/EdenConfig.h

```diff
@@ -781,23 +781,10 @@ class EdenConfig : private ConfigSettingManager {
       folly::kIsWindows ? MountProtocol::PRJFS : MountProtocol::FUSE,
       this};
 
-  // [facebook]
-  // Facebook internal
-
-  /**
-   * (Facebook Internal) Determines if EdenFS should use ServiceRouter.
-   */
-#ifdef EDEN_HAVE_SERVICEROUTER
-  ConfigSetting<bool> enableServiceRouter{
-      "facebook:enable-service-router",
-      fb_has_servicerouter(),
-      this};
-#else
```

Comment:
I don't think we define this for open source build? If it is defined we might want to fix that instead of removing this line.